waiter/src/waiter/util/utils.clj

```diff
@@ -114,6 +113,14 @@
     (log/debug "generate-secret-word" [src-id dest-id] "->" secret-word)
     secret-word))
 
+(defn- stringify-keys
+  [k]
+  (cond
+    (keyword? k) (str (.-sym k))
+    (instance? Named k) (name k)
```

Comment:
Could you delete this so symbols also fall through to the default case using `str`? It seems strange to have our keywords namespaced but the symbols not namespaced.

Reply:
Done.